blog/models.py

```diff
@@ -12,8 +15,10 @@ class Post(models.Model):
             blank=True, null=True)
 
     def publish(self):
+        """Publish post in blog."""
         self.published_date = timezone.now()
         self.save()
 
     def __str__(self):
+        """Return title of the post."""
```

Comment:
@makkaronis4e in Python, "dunder methods" (wrapped w/ double underscores, also known as "magic methods") are special methods responsible for certain behaviours of objects.
`obj.__str__()` is called if the object is being stringified, for example using `str(obj)`, meaning rendering the object into a string representation.

That said, the docstring would be smth like `Render the post instance from its title.`